less/home.less

```diff
@@ -0,0 +1,137 @@
+.home {
+  .mobile {
+    display: none;
+
+    @media @phone {
+      display: inherit;
+      width: 100%;
+    }
+  }
+
+  @media @phone {
+    .desktop {
+      display: none;
+    }
+  }
+
+  .articles {
+    padding: 40px 0;
+
+    .article {
+      .center;
+      justify-content: space-between;
+      padding: 20px 0;
+
+      @media @phone {
+        flex-direction: column;
+        padding: 0 5%;
+      }
+
+      .left {
+        padding-right: 5%;
+
+        @media @phone {
+          padding-right: 0;
+        }
+      }
+
+      .right {
+        padding-left: 5%;
+
+        @media @phone {
+          padding-left: 0;
+        }
+      }
+
+      .article-content {
+        line-height: 2.5;
+
+        .button {
+          .button(@font-color, @bt-br, @bg-color);
+          cursor: pointer;
+          padding: 10px 0;
+          margin-top: 20px;
+          width: 35%;
+          box-shadow: -3px 2px #D8D8D8;
+
+          &:hover {
+            .button(@tab-hover-color, @tab-hover-color, @tab-hover);
+            box-shadow: inset 0px 0px 0 2px #D8D8D8;
+          }
+
+          @media @phone {
+            width: 100%;
+            padding: 20px 0;
+            margin: 40px 0;
+          }
+        }
+      }
+    }
+  } // tabs end
+
+  .projects {
+
+    h3 {
+      padding-bottom: 20px;
+
+      @media @phone {
+        padding-left: 5%;
+      }
+    }
+
+    .project {
+      padding: 20px 0;
+
+      .project-title {
+        position: relative;
+
+        h2 {
+          position: absolute;
+          right: 0px;
+          bottom: -23px;
+          background-color: @rp-square;
+          padding: 28px 82px;
+
+          @media @phone {
+            width: 82%;
+            letter-spacing: 8px;
+            right: 48px;
```

Comment:
For these ones, they're not quite center. With how you have this laid out, you can set the right value to be 9% and it will center. The reason for this is that the width of your h2 is 82%, 100-82 = 18, 18 / 2 = 9. (divide by two because there's a left margin and a right margin).